scripts/install-chef-suse.sh

```diff
@@ -1166,6 +1154,18 @@ for s in xinetd dhcpd apache2 ; do
     fi
 done
 
+# activate provisioner repos
+if ! is_ses; then
+    for i in SLES12-SP1-Pool SLES12-SP1-Updates SUSE-OpenStack-Cloud-6-Pool SUSE-OpenStack-Cloud-6-Updates SLES12-SP1-HA-Pool SLES12-SP1-HA-Updates; do
+      curl --digest -u crowbar:crowbar -X POST -d "platform=suse-12.1" -d "repo=$i" http://$IP/utils/repositories/activate
```

Comment:
The admin password might not be `crowbar` (nor the username, btw). You need to use the `machine-install` user and its password (look in the script, you'll find it).